packages/app/src/request.ts

```diff
@@ -32,17 +37,32 @@ export const getProtocol = (req: Request, trust: Trust): Protocol => {
   return index !== -1 ? header.substring(0, index).trim() : header.trim()
 }
 
+/**
+ * @deprecated
+ *
+ * @param req
+ * @param trust
```

Comment:
how is it deprecated? express `req.hostname` isn't

or you want to deprecate it because getHostname already obtains the host? Either way, hostname and host are different. hostname includes only the name, aka `localhost`, but host includes port too, aka `localhost:3000`

Reply:
I marked it as deprecated because it's superceded by the `getHost` method added below - `getHost` returns an object with `hostname` and `port` properties, so the hostname can be retrieved by object destructuring
```ts
const { hostname } = getHost(...)
```

Reply:
`getHostName` is an internal method and not supposed to be used by the end user, hence marking it as deprecated is rather pointless.

Reply:
OK, I'll remove it :)